examples/client.c

```diff
@@ -1078,7 +1078,7 @@ decode_segment(const uint8_t *seg, size_t length, unsigned char *buf) {
 static int
 check_segment(const uint8_t *s, size_t length) {
 
-  size_t n = 0;
+  int n = 0;
```

Comment:
In theory, this introduces an integer overflow issue as n might become negative if length is larger than 2^31 - 1. This should not be a problem in practice (as it was not until now where we had the typecast in the return statement)..

Reply:
We only hit the issue if param `s` is at least 2G in size - extremely unlikely given that we are looking at packets - which even when fragment re-assembled (UDP) are 65536 bytes long.  I suggest it is left as-is.

Reply:
Ok, thanks.